Translator/Header/Triangle.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+#ifndef TRIANGLE_H
+#define TRIANGLE_H
+
+#include "Point.h"
+#include <fstream>
+
+class Triangle {
+private:
+    Point vertex1, vertex2, vertex3;  // Three vertices of the triangle
+
+public:
+    // Constructor and Destructor
+    Triangle(const Point& v1, const Point& v2, const Point& v3);
+    ~Triangle();
+
+    // Print function to display the triangle's vertices
+    void print() const;
```

Comment:
why are we printing things when our main motive is to return values which draws the shape